### PR TITLE
Focus 252 project feature

### PR DESCRIPTION
**Pull request by ArushiSharma95**
Project Feature implementation

### CODE REVIEW

File: frontend/src/components/dialog/DeleteDialog.js [hunk 1]
```diff
@@ -1,47 +1,43 @@
...
+                        onClick={hide}
+                        />
+                    </div>
+                    <div className="modal-body">{descriptionText}</div>
+                    <div className="modal-footer">
+                        <button
+                        type="button"
+                        className="btn btn-secondary"
+                        data-bs-dismiss="modal"
+                        show="false"
```
Comment (rbadamsjr) [inline frontend/src/components/dialog/DeleteDialog.js:32]: Is the Formatting turned on?

File: frontend/src/features/projects/ProjectsList.js [hunk 2]
```diff
@@ -1,13 +1,35 @@
 import { ProjectItem } from "features/projects";
+import {useGetConsultantProjectsQuery} from "./projectsSlice";
 
 function ProjectsList() {
+	let projectsLists=[];
+	let consultantId = 8;  //explicitly passing the userId for now (temporarily)
+	
+	const { data, isSuccess } =
+	useGetConsultantProjectsQuery(consultantId);
```
Comment (rbadamsjr) [inline frontend/src/features/projects/ProjectsList.js:9]: Add '{
		refetchOnMountOrArgChange: true,
	}' as a setting on the useGetConsultantProjectsQuery... 

i.e.

const { data, isSuccess } = useGetMyTasksQuery(userId, {
		refetchOnMountOrArgChange: true,
	});

File: frontend/src/features/projects/_project.scss [hunk 3]
```diff
@@ -5,4 +5,33 @@
...
+th,
+td {
+	border: 3px #ffff solid;
+}
+td.actions {
+	text-align: center;
+}
+span.action-item {
+	margin-left: 5px;
+	margin-right: 5px;
```
Comment (rbadamsjr) [inline frontend/src/features/projects/_project.scss:36]: Do you still need this since you defined the same values in the app.css?

**Review by jsnkle — CHANGES_REQUESTED** [review]
Again, minor changes. The rest is good stuff.

**Review by rbadamsjr — DISMISSED** [review]
Approved

**Review by jsnkle — APPROVED** [review]

**Pull request merged**